tests/runner.sh

```diff
@@ -61,7 +61,7 @@ export TEST_DBNAME
 # we strip out any output between <exclude_from_test></exclude_from_test>
 # and the part about memory usage in EXPLAIN ANALYZE output of Sort nodes
 ${PSQL} -U ${TEST_PGUSER} \
-     -v ON_ERROR_STOP=1 \
+     -v ON_ERROR_STOP=0 \
```

Comment:
I'm not sure if we should specify `ON_ERROR_STOP=1`. If we keep this line, we will not be able to test malformed input. The postmaster shuts down immediately if it has encountered any error.